style.css

```diff
@@ -28,34 +28,100 @@
     cursor: pointer;
 }
 
+.fullHistory span:hover {
+    color: gray;
+    cursor: pointer;
+}
+
+.trashButton i:hover {
+    color: red;
+    opacity: .9;
+    cursor: pointer;
+}
+
 .calcField {
-    width: 25%;
+    width: 84%;
     margin: 0 auto;
-    border-style: 1px solid;
 }
 
-.topCalc {
-    margin: 20px 0;
+.opField {
+    width: 15%;
+    margin: 0 auto;
+}
+
+.topCalc{
+    justify-content: center;
+    display: flex;
+}
+
+.output {
+    width: 40%;
+    display: flex;
+}
+
+.history {
+    width: 40%;
+    margin: 0 auto;
+    text-align: right;
+    font-size: 1em;
+    min-height: 25px;
+}
+
+.fullHistory {
+    width: 40%;
+    margin: 0 auto;
+    text-align: right;
+    font-size: 1.5em;
+    min-height: 60px;
+    border: solid 1px;
+    padding: 7px;
+}
+
+.fullHistory p {
+    margin: 5px;
+}
+
+.opEquals {
+    font-weight: bolder;
+    margin: 5px;
+}
+
+#calc{
+    text-align: right;
+    font-size: 2em;
+}
+
+#op{
+    text-align: center;
+    font-size: 2em;
 }
 
 
 @media screen and (max-width: 700px){
+    h1 {
+        font-size: 3rem;
+    }
+
+    .output {
```

Comment:
Usually when you have a bunch of elements that get the same styling it is a good idea to make a common class to use across them, maybe something that describes what they do generally